### PR TITLE
Subtree split of Form namespace, solves #9890

### PR DESCRIPTION
**Pull request by ceeram**
Implements #9890

### CODE REVIEW

File: src/Form/composer.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+    "name": "cakephp/form",
+    "description": "CakePHP Form library",
+    "license": "MIT",
+    "authors": [
+        {
+        "name": "CakePHP Community",
```
Comment (ADmad) [inline src/Form/composer.json]: Indentation is off for following 3 lines

Comment (ceeram) [inline src/Form/composer.json]: fixed now, copy pasta from the one in Validation

**Comment by markstory** [comment]
You should also update the `Makefile` so that when releases are done the read-only repo is updated.

**Comment by ceeram** [comment]
makefile also updated, forgot about that

File: src/Form/README.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    {
+        // Send an email.
+        return true;
+    }
+}
+```
+
+In the above example we see the 3 hook methods that forms provide:
+
+- _buildSchema is used to define the schema data. You can define field type, length, and precision.
```
Comment (josegonzalez) [inline src/Form/README.md]: Use backticks for the function names?

Comment (ADmad) [inline src/Form/README.md]: ..and `()` too :)

Comment (ceeram) [inline src/Form/README.md]: done and done

**Comment by jeremyharris** [comment]
This just came across my Twitter feed and I thought I'd leave a note about it here. Might make these processes easier for the team 😄

https://www.subtreesplit.com

**Comment by markstory** [comment]
I've made the new repo, it'll be populated on the next release.

**Pull request merged**